onboarding/main.go

```diff
@@ -122,3 +162,18 @@ func convertRsaPublicKeyAsPemStr(publicKey *rsa.PublicKey) (string, error) {
 
 	return string(publicKeyPem), nil
 }
+
+func getStorageClusterMetadata(ctx context.Context, operatorNamespace string, clientset *kubernetes.Clientset) (*metav1.PartialObjectMetadata, error) {
+	var storageClusterMetadata metav1.PartialObjectMetadata
+	storageClusterGVKPath := fmt.Sprintf("/apis/%s/%s/namespaces/%s/%s/%s", "ocs.openshift.io", "v1", operatorNamespace, "storageclusters", storageClusterName)
```

Comment:
1. This is a very long line, can we please reformat it?
2. Please embed the constant part into the format string. There is no need to have them as parameters.

```suggestion
	storageClusterGVKPath := fmt.Sprintf(
		"/apis/ocs.openshift.io/v1/namespaces/%s/storageclusters/%s", 
		operatorNamespace, 
		storageClusterName,
	)
```